zero/gkyl_gyrokinetic_maxwellian_correct.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
```

Comment:
Ahh, I see you renamed it do put the gyrokinetic as a prefix. Why? Maybe we should do that for all objects?